pkg/services/sqlstore/migrations/team_mig.go

```diff
@@ -24,6 +24,19 @@ func addTeamMigrations(mg *Migrator) {
 	mg.AddMigration("add index team.org_id", NewAddIndexMigration(teamV1, teamV1.Indices[0]))
 	mg.AddMigration("add unique index team_org_id_name", NewAddIndexMigration(teamV1, teamV1.Indices[1]))
 
+	mg.AddMigration("Add column uid in team", NewAddColumnMigration(teamV1, &Column{
+		Name: "uid", Type: DB_NVarchar, Length: 40, Nullable: true,
+	}))
+
+	mg.AddMigration("Update uid column values in team", NewRawSQLMigration("").
+		SQLite("UPDATE team SET uid=printf('%09d',id) WHERE uid IS NULL;").
```

Comment:
```suggestion
		SQLite("UPDATE team SET uid=printf('t%09d',id) WHERE uid IS NULL;").
```

the k8s names can not start with a number, so lets prefix these with something.  
<img width="822" alt="image" src="https://user-images.githubusercontent.com/705951/233430096-424f298c-2397-4efe-9b55-5fa97843a215.png">